packages/published-realm/.realm.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Published Workspace",
+  "backgroundURL": "https://i.postimg.cc/tgRHRV8C/pawel-czerwinski-h-Nrd99q5pe-I-unsplash.jpg",
+  "iconURL": "https://i.postimg.cc/d0B9qMvy/icon.png"
```

Comment:
I'm not using IPFS since i'm not aware of a free pinning service that we can use